### PR TITLE
Upgrade to golang 1.19/SDK 1.26

### PR DESCRIPTION
**Pull request by Sandeepyadav93**

### CODE REVIEW

**Review by abays — COMMENTED** [review]
+1 from me.  Are we waiting for other repos to merge certain PRs to upgrade to Golang 1.18 and OpSDK 1.26 before we merge this one?

**Comment by Sandeepyadav93** [comment]
@abays we only use this reusable workflow in keystone-operator which is already bumped to golang.19/sdk1.26 so this pr can be merged now.

**Review by abays — APPROVED** [review]
/lgtm

**Pull request merged**
